src/main/java/com/celements/course/CourseScriptService.java

```diff
@@ -133,4 +134,11 @@ public SpaceReference getRegistrationSpace(DocumentReference courseDocRef) {
     return null;
   }
 
+  public CourseConfirmState getConfirmeState(DocumentReference objDocRef) {
```

Comment:
- please choose better naming e.g. orderDocRef, registrationDocRef or regDocRef
- add @Null or @NotNull to all parameter and return values in public methods